strawberryfields/api/connection.py

```diff
@@ -96,10 +96,19 @@ def __init__(
         self._verbose = verbose
 
         self._base_url = "http{}://{}:{}".format("s" if self.use_ssl else "", self.host, self.port)
-        self._headers = {"Authorization": self.token}
+        self._headers = {"Authorization": self.token, "Accept-Version": self.api_version}
 
         self.log = create_logger(__name__)
 
+    @property
+    def api_version(self) -> str:
+        """The platform API version to request.
+
+        Returns:
+            str
+        """
```

Comment:
Minor: because this is a property, I think you can also write
```suggestion
        """str: the platform API version to request"""
```